src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -300,6 +300,19 @@ export class BrowsingContextProcessor {
     );
   }
 
+  async process_browsingContext_activate(
+    params: BrowsingContext.ActivateParameters
+  ): Promise<EmptyResult> {
+    const context = this.#browsingContextStorage.getContext(params.context);
+    if (!context.isTopLevelContext()) {
+      throw new InvalidArgumentException(
+        'Emulating viewport is only supported on the top-level context'
```

Comment:
nit: I know it's still a draft, but just FYI, this error message is too specific. It should not mention viewport (e.g. we could be calling `activate` for screenshot purposes).

Reply:
yeah, I just copy pasted it from the other command, still in progress.